SonarQube.Common/AnalysisProperties/AnalysisPropertyFileProvider.cs

```diff
@@ -56,9 +56,16 @@ public static bool TryCreateProvider(IEnumerable<ArgumentInstance> commandLineAr
             ArgumentInstance.TryGetArgumentValue(DescriptorId, commandLineArguments, out propertiesFilePath);
 
             AnalysisProperties locatedPropertiesFile;
-            if (TryGetPropertiesFile(propertiesFilePath, defaultPropertiesFileDirectory, logger, out locatedPropertiesFile) && locatedPropertiesFile != null)
```

Comment:
Renamed from "TryGetXXX" since it doesn't actually follow the TryGet pattern and was causing confusion.
The function returns true if no configuration errors where encountered i.e. if it's ok to continue processing. However, it doesn't mean that a properties file has necessarily been located. If the user didn't specify a settings file and we can't find the default one the locatedPropertiesFile will be null but the method will return true.